src/main/java/uk/gov/hmcts/reform/sandl/snlevents/scheduledtasks/UserTransactionsRollbackScheduledTask.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.sandl.snlevents.scheduledtasks;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.sandl.snlevents.model.db.UserTransaction;
+import uk.gov.hmcts.reform.sandl.snlevents.service.UserTransactionService;
+
+import java.util.List;
+import javax.transaction.Transactional;
+
+@Component
+@Transactional
+@EnableScheduling
+@ConditionalOnProperty(
+    prefix = "scheduler.auto-rollback",
+    name = "enabled", havingValue = "true")
+public class UserTransactionsRollbackScheduledTask {
+    private static final Logger logger = LoggerFactory.getLogger(UserTransactionsRollbackScheduledTask.class);
+
+    @Autowired
+    private UserTransactionService userTransactionService;
+
+    @Scheduled(fixedRate = 60 * 1000) // check every minute
+    public void rollbackForgottenTransactions() {
+        logger.info("Initiating rollback of forgotten transactions ");
```

Comment:
Maybe 'pending transactions' ?

Reply:
I remember of looking for a correct word, and in a group of pending vs forgotten vs timedout, the first one was stroke out, as a one that is not implicitly saying why it should be rolled back. But it was only by me, so I'm ok with changing if there are more people wanting this change

Reply:
2 is good enough, changed.